packages/nouns-contracts/contracts/StreamEscrow.sol

```diff
@@ -75,6 +75,7 @@ contract StreamEscrow is IStreamEscrow {
     ) {
         daoExecutor = daoExecutor_;
         ethRecipient = ethRecipient_;
+        require(nounsRecipient_ != address(0), 'zero address');
```

Comment:
can you please put this requirement at the top of the function?

Reply:
fixed